template/mixins/db.mixin.ts

```diff
@@ -56,7 +56,7 @@ class Connection implements Partial<ServiceSchema>, ThisType<Service>{
 	public start(){
 			if (process.env.MONGO_URI) {
 				// Mongo adapter
-				const MongoAdapter = require("moleculer-db-adapter-mongo");
+				import  MongoAdapter from "moleculer-db-adapter-mongo";
```

Comment:
![image](https://user-images.githubusercontent.com/306521/89919595-c36a0180-dbfb-11ea-8da4-6d217f665c55.png)

If you change something, please test it. `npm run dev && cd tmp && npm run dev`

Reply:
And I have no set MONGO_URI.

Reply:
even if you keep the require, it gives the error because the mongodb-adapter is not installed, so I put this part to appear only if you have using the mongo-adapter, for resolved I  add in meta.js adapter type

Reply:
okay sorry

Reply:
Ok, It works. I'm still checking with Docker/Compose.